Source/Core/DolphinQt/DolphinEmu.entitlements

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.cs.allow-jit</key>
+	<true/>
+	<!-- Needed for GameCube microphone emulation -->
+	<key>com.apple.security.device.audio-input</key>
```

Comment:
If a user isn't interested in the GameCube microphone, could they opt out of this permission? Will Dolphin not run unless it has access to the mic?

Reply:
Like any iOS or macOS application, declining permission for the microphone will not prevent Dolphin from running, just that specific function.

Reply:
This entitlement is just to declare to the system that the application uses microphone access so that it'll poke a hole in the hardened runtime. Whether the user denies permission to the mic is a separate thing.

Reply:
Got it, just confirming 😁